### ISSUE TITLE
Need Help Removing days which is not in current month

### ISSUE THREAD

**Issue by vichuer** [SimformSolutionsPvtLtd/flutter_calendar_view#141]
I need help with this amazing package. Is there a way to remove the days which is not in current month?

**Comment by PRBaraiya** [comment]
Hi, @vichuer you can provide cellBuilder property in `MonthView` and Use `FilledCell` widget that is provided by package.
this will only hide the dates that does not falls under current month. hidded date cells will still take the space.

```dart
cellBuilder: (date, events, isToday, isInMonth) {
    if(isInMonth) return SizedBox.shrink();

    return FilledCell(
        date: date,
        shouldHighlight: isToday,
        backgroundColor: Constants.offWhite,
        events: events,
        // onTileTap: onEventTap,
        // dateStringBuilder: dateStringBuilder,
    );    
},
```

**Comment by PRBaraiya** [comment]
Let me know if this helps.

**Comment by ebelevics** [comment]
I was also looking for this, but for me the problem is that week with non_month days still have cell space remained. Calendar doesn't dynamically change size if month weeks are 5 instead of 6.

**Comment by kaidrumm** [comment]
> I was also looking for this, but for me the problem is that week with non_month days still have cell space remained. Calendar doesn't dynamically change size if month weeks are 5 instead of 6.

We would also like to be able to reduce the number of weeks shown (from 6 to 5 or 4).

**Comment by apurva010** [comment]
Closing this issue as it got merged in #362.
